datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -219,13 +258,18 @@ void send(DatastreamTask task, ProducerRecord<K, V> producerRecord, Callback onC
     while (retry) {
       try {
         ++numberOfAttempt;
-        maybeGetKafkaProducer(task).ifPresent(p -> p.send(producerRecord, (metadata, exception) -> {
-          if (exception == null) {
-            onComplete.onCompletion(metadata, null);
-          } else {
-            onComplete.onCompletion(metadata, generateSendFailure(exception));
-          }
-        }));
+        Optional<Producer<K, V>> producer = maybeGetKafkaProducer(task);
```

Comment:
nit: Make this line 
```Producer<K, V> producer = maybeGetKafkaProducer(task).get()``` instead  so that the check below also reads better as producer == null.

Reply:
I want to avoid calling `get()` without `isPresent()` check. Java 11 has introduced a new api `ifPresentOrElse()` that will make this api cleaner.